dashboard/app/models/user.rb

```diff
@@ -246,6 +247,19 @@ def delete_course_as_facilitator(course)
     courses_as_facilitator.find_by(course: course).try(:destroy)
   end
 
+  def log_admin_save
+    # Do not log for development or adhoc environments.
+    return unless [:test, :staging, :levelbuilder, :production].include? rack_env
```

Comment:
Either extract the first line into something like `should_log?` and stub that in tests (preferred), or leave it as-is, and stub the `rack_env` in your specific tests to use something other than test though that might introduce side effects.